great_tables/_export.py

```diff
@@ -0,0 +1,246 @@
+
+    The `save()` method makes it easy to save a table object as an image file. The function produces
+    a high-resolution image file or PDF of the table. The output file is create by first taking a
+    screenshot of the table using a headless Chrome browser. Then, the screenshot is then cropped to
+    only include the table element. Finally, and the resulting image is saved to the specified file
+    path in the format specified (via the file extension).
+
+    Parameters
+    ----------
+    filename
+        The name of the file to save the image to.
+    path
+        An optional path to save the image to. If not provided, the image will be saved to the
+        current working directory.
+    selector
+        The HTML element selector to use to select the table. By default, this is set to "table",
+        which selects the first table element in the HTML content.
+    scale
+        The scaling factor that will be used when generating the image. By default, this is set to a
+        value of `1.0`. Lowering this will result in a smaller image, whereas increasing it will
+        result in a much higher-resolution image. This can be considered a quality setting, yet it
+        also affects the file size. The default value of `1.0` is a good balance between file size
+        and quality.
+    expand
+        The number of pixels to expand the screenshot by. By default, this is set to 5. This can be
+        increased to capture more of the surrounding area, or decreased to capture less.
+    window_size
+        The size of the window to use when taking the screenshot. This is a tuple of two integers,
+        representing the width and height of the window. By default, this is set to `(6000, 6000)`,
+        a large size that should be sufficient for most tables. If the table is larger than this
+        (and this will be obvious once inspecting the image file) you can increase the appropriate
+        values of the tuple. If the table is very small, then a reduction in these these values will
+        result in a speed gain during image capture. Please note that the window size is *not* the
+        same as the final image size. The table will be captured at the same size as it is displayed
+        in the headless browser, and the window size is used to ensure that the entire table is
+        visible in the screen capture before the cropping process occurs.
+
+    Returns
+    -------
+    None
+        This function does not return anything; it simply saves the image to the specified file
+        path.
+
+    Details
+    -------
+    We create the output file based on the HTML version of the table. With the filename extension
+    .png, we get a PNG image file. Using .pdf will yield a PDF document. The other file extensions
+    supported are .gif, .bmp, .tif, and .tiff. The file extension is used to determine the format of
+    the output file.
+
+    This process is facilitated by two libraries:
```

Comment:
```suggestion
    Details
    -------
    We create the output file based on the HTML version of the table.

    This process is facilitated by two libraries:
```


I think this is okay to remove, since the file extension determining the format is mentioned in the initial description: 

> Finally, and the resulting image is saved to the specified file path in the format specified (via the file extension).